src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double side1;
+    private double side2;
+    private double hypotenuse;
```

Comment:
```suggestion
    private double firstLeg;
    private double secondLeg;
```
нам не нужна гипотенуза, по необходимости мы можем посчитать ее из катетов (legs)